File: GitNexus/gitnexus/test/fixtures/lang-resolution/php-foreach-call-expr/Repo.php
<?php

class Repo {
    public string $name;

    public function __construct(string $name) {
        $this->name = $name;
    }

    public function save(): void {}
}

/**
 * @return Repo[]
 */
function getRepos(): array {
    return [new Repo("main")];
}

File: GitNexus/gitnexus/test/fixtures/lang-resolution/java-foreach-call-expr/models/Repo.java
package models;

import java.util.List;

public class Repo {
    private String name;

    public Repo(String name) {
        this.name = name;
    }

    public void save() {}

    public static List<Repo> getRepos() {
        return List.of(new Repo("main"));
    }
}
